extendeddatacrossword_test.go

```diff
@@ -2,6 +2,7 @@ package rsmt2d
 
 import (
 	"bytes"
+	crand "crypto/rand"
 	"errors"
 	"fmt"
 	"math/rand"
```

Comment:
[no change needed] in a follow up PR we may consider replacing usage of `math/rand` with `crypto/rand` and then deleting the import alias for `crypto/rand`

Reply:
Unfortunately, `crypto/rand` lacks handy IntN funciton, only 
```
func Int(rand io.Reader, max *big.Int) (n *big.Int, err error) 
```
While `math/rand` lacks ability to generate random []byte slice. I wish one day there will be single package for both

Reply:
There is rand v2